gen/suites/actor_creation/create_actor_failures.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+
+	"github.com/filecoin-project/test-vectors/chaos"
+	. "github.com/filecoin-project/test-vectors/gen/builders"
+)
+
+func createAccountActorWithExistingAddr(v *Builder) {
+	v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+	alice := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+	bob := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+	v.CommitPreconditions()
+
+	balanceBefore := v.Actors.Balance(alice.Robust)
+	msg := v.Messages.Raw(alice.ID, chaos.Address, chaos.MethodCreateAccountActorWithAddr, MustSerialize(&bob.Robust), Nonce(0), Value(big.Zero()))
+	v.CommitApplies()
+
+	v.Assert.EveryMessageResultSatisfies(ExitCode(exitcode.SysErrorIllegalArgument))  // make sure that we get SysErrorIllegalArgument error code
+	v.Assert.BalanceEq(alice.Robust, big.Sub(balanceBefore, CalculateDeduction(msg))) // make sure that gas is deducted from alice's account
+}
+
+func createUnknownActor(v *Builder) {
+	v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+	alice := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+	v.CommitPreconditions()
+
+	balanceBefore := v.Actors.Balance(alice.Robust)
+	msg := v.Messages.Raw(alice.ID, chaos.Address, chaos.MethodCreateUnknownActor, MustSerialize(&alice.Robust), Nonce(0), Value(big.Zero()))
```

Comment:
I am not happy with this test, because I am not sure what kind of params to send to `chaos.MethodCreateUnknownActor` (I got errors that params are required). Not sure how to signal `no/empty params` just yet, so I am passing `alice.Robust` which is not really used.

Furthermore if there is a bug in the code under test and it doesn't abort on a condition checking for the actor code, it would definitely abort on this address, which is not great.

Reply:
Try `adt.Empty` for no param.s